device/acpi_shutdown_device.go

```diff
@@ -0,0 +1,52 @@
+package device
+
+import "log"
+
+const (
+	ACPIShutDownDevPort = uint64(0x600)
```

Comment:
Since this device does not appear to be used in #148, how about simply removing it?
If we need it, could you provide a link to the ACPI DSDT table spec?
I do not know where 0x600 is the number from.

Reply:
This device is not really in the spec, but....

EDK2/CloudHv uses this IO-Port for letting the VM know about a shutdown.

[See here](https://github.com/cloud-hypervisor/edk2/blob/ch/OvmfPkg/Include/IndustryStandard/CloudHv.h)

It's a requirement by the EDK2-Adaption. Even though we're not using it yet, I would like to keep it for the case if someone*tm wants to implement the host side of handling the event. Maybe also relevant for: #87

Reply:
OK. Thanks for the clarification. I'll leave it as is then.